logging/google/cloud/logging/logger.py

```diff
@@ -436,29 +462,34 @@ def log_proto(self, message, labels=None, insert_id=None, severity=None,
         """
         self.entries.append(
             ('proto', message, labels, insert_id, severity, http_request,
-             timestamp))
+             timestamp, resource))
 
-    def commit(self, client=None):
+    def commit(self, client=None, resource=None):
```

Comment:
I moved it to the constructor which I think makes more sense for a ContextManager.